pkg/gke/components/__tests__/Networking.test.ts

```diff
@@ -135,7 +135,7 @@ describe('gke Networking', () => {
 
     wrapper.setProps({ network: 'host-shared-vpc' });
     await wrapper.vm.$nextTick();
-    expect(wrapper.emitted('update:subnetwork')?.[0]?.[0]).toBe('projects/host-project-309915/regions/us-west1/subnetworks/host-shared-vpc-us-west1-subnet-public');
+    expect(wrapper.emitted('update:subnetwork')?.[0]?.[0]).toBe('projects/host-project-309915/global/networks/host-shared-vpc');
```

Comment:
I think this is indicative of something wrong with the networking code. I would still expect the emitted event here to refer to a subnetwork selfLink, since one was defined previously and mock data hasn't changed, but that's a network selfLink. Looking at gcp mock data, sn.subnetwork is defined when the user has selected a shared VPC. I think https://github.com/rancher/dashboard/issues/16052 is not needed in light of that.